pkg/tfbridge/schema.go

```diff
@@ -342,7 +344,7 @@ func MakeTerraformInputs(
 
 // makeSingleTerraformInput converts a single Pulumi property value into a plain go value suitable for use by Terraform.
 // makeSingleTerraformInput does not apply any defaults or other transformations.
-func makeSingleTerraformInput(
+func makeSingleTerraformInputForSetElement(
```

Comment:
This is not a great name as it can be used on anything right? 

Perhaps leave the name as it was and comment that it will resurrect schema.Set values?

Reply:
I'd move this to detailed_diff.go also maybe as that's the only caller.